app/src/main/java/ch/epfl/sweng/swenggolf/notification/NotificationsActivity.java

```diff
@@ -77,28 +81,31 @@ private void setRecyclerView(View inflated) {
 
         checkUserPoints();
 
-        fetchNotifications(inflated);
+        fetchNotifications();
 
-        if (mAdapter.getItemCount() == 0) {
-            inflated.findViewById(R.id.message_empty).setVisibility(View.VISIBLE);
-        }
     }
 
-    private void fetchNotifications(final View inflated) {
+    private void fetchNotifications() {
         User currentUser = Config.getUser();
         ValueListener<List<Notification>> listener = new ValueListener<List<Notification>>() {
```

Comment:
You can define this listener outside of the method to resolve the Code Climate issue.

Reply:
Done